source/fileio/read_network.py

```diff
@@ -106,6 +106,10 @@ def read(self, flownetwork):
                     edge_list[eid, 1] = ii_br
                     eid += 1
 
+        edge_list = np.append(edge_list, [[5, 14]], axis=0)
+        edge_list = np.append(edge_list, [[13, 22]], axis=0)
+        nr_of_edges += 2
+
```

Comment:
Why do we need these lines of code (109-111)?
Did you use them to test a special case? If so, can you please remove them?